evap/staff/tests/test_views.py

```diff
@@ -1953,6 +1980,20 @@ def test_textanswers_full_view(self):
             page = self.app.get(self.url + '?view=full', user=self.manager, status=200)
             self.assertContains(page, self.answer)
 
+    def test_exclude_unreviewed_evaluations(self):
```

Comment:
exclude unreviewed evaluations? Shouldn't it be something like "exclude unfinished evaluations"?